conf/messages

```diff
@@ -317,6 +320,10 @@ destinationType.error.required.import = Select the place of destination for this
 destinationType.checkYourAnswersLabel = Destination type
 destinationType.change.hidden = destination type
 
+destinationWarehouse.title = Where is the destination warehouse located?
+destinationWarehouse.heading = Where is the destination warehouse located?
+destinationWarehouse.error.required = Select the location of the destination warehouse
```

Comment:
Not needed

Reply:
Fixed in [0bc651d](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/50/commits/0bc651dcc4922eb6775656fd5aa44f25409f29fb)